stdlib/2and3/logging/__init__.pyi

```diff
@@ -18,6 +18,7 @@ else:
     _ExcInfoType = Union[None, bool, _SysExcInfoType]
 _ArgsType = Union[Tuple[Any, ...], Dict[str, Any]]
 _FilterType = Union['Filter', Callable[['LogRecord'], int]]
+_Level = Union[int, str, Text]
```

Comment:
"Text" always includes "str", in both Python 2 and Python 3. So can you just write `Union[int, Text]`?